### DISCUSSION TITLE
BloatyNosyApp 0.50

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#35] [Announcements]
- This update brings numerous optimizations to the core of the app and a more compact UI.

![BloatyNosy_tJZqO8FWza](https://user-images.githubusercontent.com/57478606/234258624-c0ecc7cd-41e1-492c-bd69-15591a8eb89a.png)

Thank you very much for the lively interest in the store version and the numerous ratings and feedback.

**Please note that the version published on GitHub is more up-to-date than the one in the Microsoft store. Updates in the store must always be approved by Microsoft, which can take up to three days.**

[<img alt="alt_text" width="240px" src="https://upload.wikimedia.org/wikipedia/commons/f/f7/Get_it_from_Microsoft_Badge.svg" href=""/>](https://apps.microsoft.com/store/detail/XPDCJG88N6FD16) 
_Microsoft Store deep link: ms-windows-store://pdp/?productid=XPDCJG88N6FD16_

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/BloatyNosy/releases/tag/0.50.0'>BloatyNosyApp 0.50</a>.</em>

**Comment by unitedonline** [comment]
Hello,
I scanned the zip file on the VirusTotal website and found - 
MaxSecure - Trojan.Malware.300983.susgen

Can you please confirm?

**Comment by Belim** [comment]
Yes, I can confirm that it is a false positive.

**Reply by unitedonline** [reply]
Thank you for your immediate reply.
